src/sentry/south_migrations/0386_auto__add_field_groupenvironment_first_release_id__add_index_groupenvi.py

```diff
@@ -0,0 +1,1065 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Adding field 'GroupEnvironment.first_release_id'
+        db.add_column('sentry_groupenvironment', 'first_release_id',
+                      self.gf('sentry.db.models.fields.bounded.BoundedPositiveIntegerField')(null=True),
+                      keep_default=False)
+
+        # Adding index on 'GroupEnvironment', fields ['environment_id', 'first_release_id']
+        db.create_index('sentry_groupenvironment', ['environment_id', 'first_release_id'])
```

Comment:
Definitely need to change this index to raw SQL for Postgres so we can do `CREATE INDEX CONCURRENTLY`.

Reply:
I'll mark it as dangerous, do you need any other changes here?

Reply:
Or do want it to just be executed directly via the cursor…?

Reply:
Like this: https://github.com/getsentry/sentry/blob/master/src/sentry/south_migrations/0359_auto__add_index_tagvalue_project_id_key_last_seen.py#L16-L27

Reply:
Updated here: 45c3d4d650b72021251e91fcaf93e2023015522d

I think this makes it so the migration could technically get into a half-applied state, but it looks like we do it elsewhere (e.g. https://github.com/getsentry/sentry/blob/3a9695bd223082de497b725aa62b4571f08b5e8d/src/sentry/south_migrations/0341_auto__add_organizationintegration__add_unique_organizationintegration_.py) so I didn't do anything about it